react-native-cli/index.js

```diff
@@ -198,7 +199,14 @@ function run(root, projectName, logLevel) {
   if (logLevel === 'debug' || logLevel === 'verbose') {
     spawnArgs = {stdio: 'inherit'};
   }
-  var proc = spawn('npm', args, spawnArgs);
+  var proc;
+  if (os.platform() === 'win32'){
+    args.unshift('npm');
+    args.unshift('/c');
+    proc = spawn('cmd', args, spawnArgs);
```

Comment:
Is this `cmd /c npm install --save react-native`? I've never seen that syntax, can you add some comment explaining what it does?